optimade/validator/utils.py

```diff
@@ -161,7 +161,9 @@ def add_failure(
 
 
 class Client:  # pragma: no cover
-    def __init__(self, base_url: str, max_retries=5):
+    def __init__(
+        self, base_url: str, max_retries: int = 5, headers: Dict[str, str] = None
+    ):
```

Comment:
For completion - not necessary.
```suggestion
    ) -> None:
```

Reply:
Also, shouldn't you update the doc strings here for the API Reference documentation?